flags.go

```diff
@@ -583,9 +622,15 @@ func validateFlags(flags *flagStorage) (err error) {
 
 	if !flags.ClientProtocol.IsValid() {
 		err = fmt.Errorf("client protocol: %s is not valid", flags.ClientProtocol)
+		return
```

Comment:
@kislaykishore Once again, I've just followed the conventions in the code. Technically there is nothing wrong with such returns. I'm making changes now for making forward progress, but it needs to be a part of coding guidelines discussion, rather than in a one-off code review.